### DISCUSSION TITLE
is Hstore supported by bun?

### DISCUSSION THREAD

**Discussion by OmerNinburg-RecoLabs** [uptrace/bun#304] [Q&A]
I just migrated from [go-pg](https://github.com/go-pg/pg) and I am getting some errors on inserts from the following model:
```
type Model struct {
	ID     uuid.UUID         `bun:"id,pk,type:uuid,default:uuid_generate_v4()"`
	Hstore map[string]string `bun:"store,hstore"`
	Name   string            `bun:"name,unique"`
}
```

the resulting insert query generated by bun for some dummy data is:
```
INSERT INTO "models" ("id", "store", "name") VALUES ('e882d862-686c-4359-8e06-a22c8df32ee3', '{"a":"a","b":"b","c":"c"}', 'name')
```
which is not a valid insert query for a hstore field

After looking in the bun git repo - I couldn't find the Hstore scanner and appender - does this mean bun does not support this?
Do I have any other way to migrate to bun without chaning my model?

**Comment by vmihailenco** [comment]
See https://github.com/uptrace/bun/issues/291
